src/udev/mtd_probe/mtd_probe.c

```diff
@@ -27,31 +28,34 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#include "alloc-util.h"
+#include "fd-util.h"
 #include "mtd_probe.h"
 
-int main(int argc, char** argv)
-{
-        int mtd_fd;
-        int error;
+int main(int argc, char** argv) {
+        _cleanup_close_ int mtd_fd = -1;
         mtd_info_t mtd_info;
+        int error;
```

Comment:
I prefer `r`.

Reply:
me too, but didnt want to refactor too much here, only the stuff that I think was super ugly...

But let's fix this at the same time, I'll force push a new version with this renamed shortly.